formatencoder.go

```diff
@@ -153,7 +155,14 @@ func getValue(buf *buffer.Buffer, key string) (interface{}, bool) {
 	switch dataType {
 	case jsonparser.NotExist:
 		return nil, false
-	case jsonparser.Array, jsonparser.Boolean, jsonparser.Null, jsonparser.Number, jsonparser.Object, jsonparser.String, jsonparser.Unknown:
+	case jsonparser.String:
+		if !unescapeStrings {
+			return value, true
+		} else {
+			str, _ := jsonparser.ParseString(value)
+			return str, true
+		}
```

Comment:
This can be shortened like this:
```suggestion
		if !unescapeStrings {
			return value, true
		}
		return jsonparser.ParseString(value), true
```

Reply:
Change done
Though we can't return jsonparser.ParseString() result directly, as it is a tuple (str, err)

Reply:
My bad 🤦🏻‍♂️ Thank you for catching it